force-app/test/ADDR_Addresses_TEST.cls

```diff
@@ -2560,6 +2560,7 @@ public with sharing class ADDR_Addresses_TEST {
                 MailingStreet__c, MailingCity__c, MailingState__c,
                 MailingPostalCode__c, MailingCountry__c
             FROM Address__c
+            ORDER BY Name DESC
```

Comment:
@lparrott what was the reason for adding the order by here?

Reply:
Without this, one of the tests was failing. We originally were ordering by MalingState, but had to remove that for encryption reasons. This ensures the order of Addresses for testing purposes, but we could figure out a way to remove it.